demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -86,7 +87,7 @@ def __new__(
         return model_cls
 
 
-class BaseContent(ABC, BaseModel, metaclass=BaseContentMetaclass):
+class BaseContentModel(ABC, BaseModel, metaclass=BaseContentMetaclass):
```

Comment:
let's rename:
This base class will be `BaseContent`
The subclass will be `BaseContentWithPath`

@dantavori what do you think?
The idea is to create a class between `BaseContent` and the Pack/Content (without commands), so we can use the `path` attribute